### PR TITLE
Be create dockerimage

### PR DESCRIPTION
**Pull request by amroakmal**
Create Dockerfile for BE

### CODE REVIEW

**Comment by shakram02** [comment]
please rebase this branch as it can't be merged if it has conflicts. :v:

**Comment by shakram02** [comment]
please resolve

**Review by shakram02 — APPROVED** [review]

File: graderx/views.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 ALLOWED_EXTENSIONS = {'rar', '7z', 'zip'}
 
+currentStatusString = ""
+
```
Comment (amohamed97) [inline graderx/views.py]: currentStatusString is already in manager.py

File: graderx/graders/manager.py [hunk 2]
```diff
@@ -71,9 +74,18 @@ def run_grader_commands(lab_id):
...
+def get_status():
+    return currentStatusString
+
+
 def run_grader(lab_id: str, submissions_file: FileStorage) -> dict:
+    global currentStatusString
     curr_dir = str(Path(__file__).parent.resolve())
+    currentStatusString = "Processing"
     extract_submissions(Path(f"{curr_dir}/courses/cc451/app/{lab_id}/submissions/2020"), submissions_file)
+    time.sleep(1)
```
Comment (amohamed97) [inline graderx/graders/manager.py]: is this required or was it for demo purposes only?

**Review by amohamed97 — APPROVED** [review]

**Pull request merged**